### DISCUSSION TITLE
Can't open server on Mac

### DISCUSSION THREAD

**Discussion by bringerovdeath** [apohl79/audiogridder#1238] [Help]
The server *is* running because it's functioning with the plugin inside of Cubase, but when I try to open the server app to adjust settings, nothing happens, it doesn't open, and there are no menus that pop up. What am I doing wrong here?

I'm on a 2019 Macbook Pro, OSX 10.14.6.

**Comment by oldtos** [comment]
Hmmm on a pc you right click the sever icon in system tray...
